source/intro.tex

```diff
@@ -60,6 +60,9 @@
 \item ISO 80000-2:2009, \doccite{Quantities and units ---
 Part 2: Mathematical signs and symbols
 to be used in the natural sciences and technology}
+\item The Unicode Consortium. Unicode Standard Annex, UAX \#29, \doccite{Unicode Text Segmentation} [online].
+Edited by Mark Davis. Revision 35. 2019-02-15 [viewed 2020-02-23].
```

Comment:
Maybe say "Revision 35 for Unicode 12.0.0" to bring in the Unicode version number as well?

Reply:
I had the version in there 2 tries before and Hubert had me to take it out.  It doesn't help that the "Unicode Text Segmentation" section says it's version 12.0.0 while the document itself says it's 12.1.0.

@hubert-reinterpretcast do you agree to putting the version back in as Jens suggests?

Reply:
"Revision 35; issued for Unicode 12.0.0"?

Reply:
applied Hubert's suggestion above